src/applications/vaos/referral-appointments/ScheduleReferral.jsx

```diff
@@ -93,27 +40,33 @@ export default function ScheduleReferral() {
             with a community care provider.
           </p>
         </va-additional-info>
-        <ReferralAppLink linkText="Schedule your appointment" />
+        <ReferralAppLink
+          linkText="Schedule your appointment"
+          id={currentReferral.UUID}
+        />
         <h2>Details about your referral</h2>
```

Comment:
Should we write a test for the date format or the va addtional info or all the things? I'm not sure where to draw there. It probably already has 100% test coverage so it's probably fine.

Reply:
There is a ticket for coverage of this component already, its a bit outside of the scope of this ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/95270